c/datasetService.c

```diff
@@ -99,13 +99,36 @@ static int serveDatasetContents(HttpService *service, HttpResponse *response){
     respondWithDataset(response, filename, TRUE);
   }
   else if (!strcmp(request->method, methodPOST)){
+    /* this section is WIP to allow saving of members when another user
+       has the directory open */
     char *l1 = stringListPrint(request->parsedFile, 1, 1, "/", 0);
     char *percentDecoded = cleanURLParamValue(response->slh, l1);
-    char *filenamep1 = stringConcatenate(response->slh, "//'", percentDecoded);
-    char *filename = stringConcatenate(response->slh, filenamep1, "'");
+    // char *datasetName = cleanURLParamValue(response->slh, l1);
+    // /* update uses USS filename convention to refer to MVS dataset */
+    char *filenamep1 = stringConcatenate(response->slh, "//'", percentDecoded);     /* prepend  //'  to DSN*/
+    // char *filenamep1 = stringConcatenate(response->slh, "~/", datasetName);      /* prepend  ~/  to filename */
+    char *filename = stringConcatenate(response->slh, filenamep1, "'");             /* append   '    to DSN */
+
+    /* append PID to filename */
+    pid_t pid = getpid();
+    char pid_string[10];
```

Comment:
Please use the style the rest of the code uses. That is, use camel case.

Reply:
code deleted